documentation/restore.md

```diff
@@ -0,0 +1,26 @@
+# Restore Puppet Enterprise using the PEADM module
+
+Once you have a [backup](backup.md) you can restore a PE primary server. It is important to highlight that you can not use the `peadm::restore` plan with a backup that was not created with the `peadm::backup` plan.
+
+As in the `peadm::backup` plan, you can choose what you want to restore by specifying the parameter `restore`
+
+Example:
+
+```
+# restore_params.json
+
+{
+  "restore": {
+    "orchestrator": false,
+    "puppetdb": false,
+    "rbac": true,
+    "activity": false,
+    "ca": false,
+    "classifier": false,
+  },
+  "input_file": "/tmp/path"
```

Comment:
Where does `path` need to point to? A tarball, a directory?
One would hope the restore would work with exactly the same parameter set as backup, so maybe use the same parameter "backup_path" instead of "destination" or "input_file"?

Reply:
@timidri this was somewhat debated with Reid in the original review, I had originally used backup_path but this was viewed as potentially confusing and changed to input_file which should be the full path tarball file

Reply:
Hmmm. OK, in that case, I think we should mention that "input_file" should actually point to the tarball in the example: 
```
  "input_file": "/tmp/path/backup_tarball.tgz"
```